### DISCUSSION TITLE
How can I configure swc to enable core-js?

### DISCUSSION THREAD

**Discussion by jlachniet** [parcel-bundler/parcel#7195] [Q&A]
Hello, I want to configure swc so that parcel builds with core-js polyfills. I tried creating a .swcrc file with the following:

```
{
	"env": {
		"mode": "entry",
		"coreJs": 3
	}
}
```

When I call `parcel build`, I get no errors or warnings, but I don't see any effect. I have my `browserslist` set in my `package.json` and I can see that arrow functions are correctly transformed into regular functions, but if I include an array map, it is not transformed to include the core-js polyfill.

As far as I can tell, my `.swcrc` config is not actually being used by parcel. If I modify my config to include invalid JSON I don't get an error, and no other options in the config seem to make a difference.

How can I make swc use this config and include core-js polyfills?

**Comment by keithlayne** [comment]
https://github.com/parcel-bundler/parcel/blob/62f64cae33a7bc1d2f5117472f9fd8bc2a931ab0/packages/transformers/js/core/src/lib.rs#L38

Since swc is used as a lib...maybe that's not currently an option.
